src/main/java/com/mongodb/jdbc/MongoSQLDatabaseMetaData.java

```diff
@@ -20,6 +38,11 @@ public MongoSQLDatabaseMetaData(MongoConnection conn) {
         super(conn);
     }
 
+    private BsonDocument createBottomBson(BsonElement... elements) {
+        BsonDocument bot = new BsonDocument(Arrays.asList(elements));
+        return new BsonDocument(BOT_NAME, bot);
+    }
```

Comment:
This is introduced in #55 and I'll probably end up rebasing on top of that. Figured I'd get a head start using it here.